src/AllMenu/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+
+class AllMenu extends Component {
+  constructor(props){
+    super(props)
+
+    this.state = {
+      dish_name:'',
+      description:''
+    }
+  }
+
+  toggleChangeMJ(){
+     this.setState(prevState => ({
+       isMJ: !prevState.isMJ,
```

Comment:
what is MJ and JB?

Reply:
it was example from some tutoril how to save text from checkbox, so i deleted it